CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+## 1.3.2
+
+### Added
+* Allow direct application of standard or flexible plugin ([#144](../../issues/144))
```

Comment:
They're relative links. They work in the branch and also after being merged into master for the current repository. Would you prefer full links?